kube-client/src/api/entry.rs

```diff
@@ -415,7 +472,7 @@ mod tests {
             .get_or_insert_with(BTreeMap::default)
             .insert("key".to_string(), "value3".to_string());
         assert!(
-            matches!(entry2.commit().await, Err(Error::Api(ErrorResponse{reason,..})) if reason == "Conflict")
+            matches!(entry2.commit().await, Err(CommitError::Save(Error::Api(ErrorResponse { reason, .. }))) if reason == "Conflict")
```

Comment:
you could put a pub method on `CommitError` if you want an easier way to determine this maybe? :thinking:

Reply:
I'd rather not. Any actual users should just fail and retry at a level where it makes sense.